server/src/main/kotlin/suwayomi/tachidesk/manga/impl/update/UpdateStatus.kt

```diff
@@ -0,0 +1,35 @@
+package suwayomi.tachidesk.manga.impl.update
+
+import mu.KotlinLogging
+import suwayomi.tachidesk.manga.model.dataclass.MangaDataClass
+
+var logger = KotlinLogging.logger {}
+class UpdateStatus {
```

Comment:
Maybe make the default constructor a constructor with fields since your just assigning it anyway.